lib/extensions.js

```diff
@@ -359,6 +366,29 @@ function getVersionInfo(binding) {
   ].join(eol);
 }
 
+/**
+ * Gets the platform variant, currently either an empty string or 'musl' for Linux/musl platforms.
+ *
+ * @api public
+ */
+
+function getPlatformVariant() {
+  if (process.platform !== 'linux') {
+    return '';
+  }
+  var contents = '';
+  try {
+    contents = fs.readFileSync(process.execPath);
+    if (!contents.indexOf) {
```

Comment:
Were you able to figure out how contents wasn't a string or buffer? This intent of line isn't clear.